docs/data/base/components/switch/UseSwitchesBasic.tsx

```diff
@@ -76,27 +87,32 @@ const BasicSwitchInput = styled('input')`
   margin: 0;
 `;
 
-const BasicSwitchThumb = styled('span')`
+const BasicSwitchThumb = styled('span')(
+  ({ theme }) => `
   display: block;
   width: 16px;
   height: 16px;
-  top: 4px;
+  top: 3px;
   left: 4px;
```

Comment:
```suggestion
  left: 3px;
```

Adjusting it here due to the reduced track size!